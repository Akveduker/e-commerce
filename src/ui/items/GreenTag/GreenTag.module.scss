@import "../../../styles/_vars.scss";
@import "../../../styles/_mixins.scss";
.tag {
    @include fontSemibold;
    color: $C2A;
    background-color: $C2E;
    border-radius: $borderRadius;
    padding: 0 8px;
}
