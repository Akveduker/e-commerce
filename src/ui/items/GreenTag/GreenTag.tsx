import React, { FC } from 'react';
import s from './GreenTag.module.scss'
interface GreenTagProps {
    children: React.ReactNode;
}
const GreenTag: FC<GreenTagProps> = ({ children }) => {
    return (
        <div className={s['tag']}>
            {children}
        </div>
    );
};

export default GreenTag;