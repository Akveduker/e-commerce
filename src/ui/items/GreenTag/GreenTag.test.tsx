import { render, screen } from "@testing-library/react"
import GreenTag from "./GreenTag"

describe('GreenTag', () => {
    test('Снапшот', () => {
        const { container } = render(
            <GreenTag>
                tag
            </GreenTag>
        )
        expect(container).toMatchSnapshot()
    })
    test('Есть в документе', () => {
        render(
            <GreenTag>
                tag
            </GreenTag>
        )
        expect(screen.queryByText('tag')).toBeInTheDocument()
    })
})