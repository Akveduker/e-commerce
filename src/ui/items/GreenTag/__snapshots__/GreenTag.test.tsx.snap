// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreenTag Снапшот 1`] = `
<div>
  <div
    class="tag"
  >
    tag
  </div>
</div>
`;
